specifyweb/frontend/js_src/lib/components/Router/RouterUtils.tsx

```diff
@@ -80,12 +81,12 @@ export const toReactRoutes = (
 
     const resolvedElement =
```

Comment:
I noticed a bug here (was preseent before)
If rawElement is a component, not a function, then `title` is ignored
(line 95 may need to be modified to include title too)
in that case, factor our AppTitle ternary into a common variable to not duplicate it

Reply:
fixed